packages/apps-config/src/api/spec/snowbridge.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright 2017-2021 @polkadot/apps-config authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { definition } from '@snowfork/snowbridge-types';
+
+export default definition;
```

Comment:
Shouldn't this be `{ bundle }`? (i.e. these are bundle exports - have not looked at the code, only the npm package page)

Reply:
... ignore me, looked at the code :)